manifests/init.pp

```diff
@@ -83,7 +83,7 @@
     Class['passenger::config']
   }
 
-  if type($include_build_tools) == 'string' {
+  if type3x($include_build_tools) == 'string' {
```

Comment:
I don't think `type3x()` or `is_string()` are going anywhere, though puppet 4 patterns don't need either of them. And either function is just one side of the same coin.